react/src/App.jsx

```diff
@@ -98,6 +108,13 @@ function App() {
     };
   }, []);
 
+  // @NOTE: We do not recommend adding events dynamically (on button clicks). Instead, we prefer to add them
+  // when the component mounts, and remove them when the component unmounts.
+  // However, if you still want to add events to button clicks, you can do so using the useCallback hook.
+  // just like we have used it below.
+
+  // @NOTE: In order to add Message Add event handler, add the following code.
```

Comment:
Add another sentence here to explain that we don't recommend attaching event listeners this way, but if a particular use case calls for it, then this is the right way.

Reword this appropriately ^